doc/btp_gap.txt

```diff
@@ -637,6 +637,180 @@ Commands and responses:
 
 		In case of an error, the error response will be returned.
 
+	Opcode 0x2a - Create Advertising Set
+
+		Controller Index:	<controller id>
+		Command parameters:	<none>
+		Return Parameters:	Created Advertising Set ID (1 octet)
+
+		This command is used to create an advertising set, which can be used
+		for both legacy and extended advertisements. The set is created in
+		the stopped state.
+
+		The set is created with the default parameters of:
+			SID (1 octet) = 0
+			Address Type (1 octet) = 0x00 (Identity)
+			Advertising Type (1 octet) = 0x00 (Legacy Not Connectable, Not Scannable)
+			Filter Accept List (1 octet) = 0x00 (Do not use the filter accept list)
+			Directed Advertising (1 octet) = 0x00 (Do not use directed advertising)
+			Options (2 octets) = 0x0000
+			Minimum Interval (4 octets) = 160 (100 ms)
+			Maximum Interval (4 octets) = 320 (200 ms)
+			Peer Address Type (1 octet) = 0x00 (Public)
+			Peer Address (6 octets) = 00:00:00:00:00:00
+
+		The parameters can be changed with the Update Advertising Set
+		Parameters command.
+
+		The set can be deleted with Delete Advertising Set.
+
+		The return value is the ID of the newly created set.
+
+		Note that the state of advertising sets created with this command do
+		not affect the Current_Settings state.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x2b - Delete Advertising Set
+
+		Controller Index:	<controller id>
+		Command parameters:	Advertising Set Id (1 octet)
+		Return Parameters:	<none>
+
+		This command is used to delete a previously created advertising set.
+		If the advertising set is currently active, it will be stopped.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x2c - Update Advertising Set Parameters
+
+		Controller Index:	<controller id>
+		Command parameters:	Advertising Set Id (1 octet)
+					SID (1 octet)
+					Address Type (1 octet)
+					Advertising Type (1 octet)
+					Filter Accept List (1 octet)
+					Directed Advertising (1 octet)
+					Options (2 octets)
+					Minimum Interval, in 0.625ms units (4 octets)
+					Maximum Interval, in 0.625ms units (4 octets)
+					Peer Address Type (1 octet)
+					Peer Address (6 octets)
+		Return Parameters:	<none>
+
+		This command is used to configure the parameters of a previously created
+		advertising set. The advertising set must be stopped when the command
+		is given, otherwise an error will be emitted.
+
+		The allowed values for SID are 0...15.
+
+		The allowed values for Address Type are:
+			0x00 = Identity Address
+			0x01 = Private Address (resolvable if connectable)
+
+		The allowed values for Advertising Type are:
```

Comment:
I'd prefer to have this defined as flags, not values (easier to extend if needed), eg:
connectable, scannable, legacy_pdu, directed (although this could be implicit based on peer addr), anonymous, etc

also some options seems to be to zephyr's API specific (eg disabling 2M phy or using device name)